index.html

```diff
@@ -33,28 +33,212 @@ <h1 class="headline">
             </p>
 
             <ul class="social-media-icons" >
-                <li>
-                    <a href="https://twitter.com"><img class ="icon" src="images/twitter.svg" alt="twitter"></a>
+                <li class ="social-icon">
+                    <a href="https://twitter.com"><img src="images/twitter.svg" alt="twitter"></a>
                 </li>
-                <li>
-                    <a href="https://linkedin.com"><img class ="icon" src="images/Linkedin_icon.svg" alt="LinkedIn"></a>
+                <li class ="social-icon">
+                    <a href="https://linkedin.com"><img src="images/Linkedin_icon.svg" alt="LinkedIn"></a>
                 </li>
-                <li>
-                    <a href="https://medium.com"><img class ="icon" src="images/medium.svg" alt="Medium"></a>
+                <li class ="social-icon">
+                    <a href="https://medium.com"><img src="images/medium.svg" alt="Medium"></a>
                 </li>
-                <li>
-                    <a href="https://github.com"><img class ="icon" src="images/github.svg" alt="github"></a>
+                <li class ="social-icon">
+                    <a href="https://github.com"><img src="images/github.svg" alt="github"></a>
                 </li>
-                <li>
-                    <a href="https://angellist.com"><img class ="icon" src="images/angellist.svg" alt="angellist"></a>
+                <li class ="social-icon">
+                    <a href="https://angellist.com"><img src="images/angellist.svg" alt="angellist"></a>
                 </li>
             </ul>
           
             <button type="button" class="button-style">
                 <i class="fa fa-chevron-down" aria-hidden="true"></i>
             </button>
+        </section>
+
+        <section class="recent-works-container">
+            <h1 class="headline">
+                My Recent Works
+            </h1>
```

Comment:
Kindly change the heading for each project for any heading from `h2` to `h6`, for example, `h2`. the `h1` tag should only exist once on your page